contributing_to_docs/doc_guidelines.adoc

```diff
@@ -743,21 +743,66 @@ Replace all public IP addresses with an address from the following blocks. These
 
-* All links to internal content is created using `xref` and **must have an anchor ID**.
+* All links to internal content is created by using an `xref`, and each `xref` **must have an anchor ID**.
 * Only use `xref` in assemblies, not in modules.
-* All links to external websites are created using `link`.
-* Links between different distros, such as from ROSA to OpenShift Container Platform, are external links and not cross-references. These external links use `link` instead of `xref` and need to be in distro-specific files or conditionalized to apply to the relevant distros.
+* All links to external websites are created by using the `link` attribute.
+* Links between different distributions, or _distros_, such as from ROSA to OpenShift Container Platform, are external links and not cross-references. These external links use `link` instead of `xref` and must be in distro-specific files or a conditionalized state to apply to the relevant distros.
 
 [IMPORTANT]
 ====
-Do not split link paths across lines when wrapping text. This will cause issues with the doc builds.
+Do not split link paths across lines when wrapping text. This will cause issues with the documentation builds.
 ====
 
+[id="lead-in-link-sentences"]
+=== Lead-in link sentences
+
+Using a lead-in link sentence that adheres to guidelines maintains consistency across OpenShift Container Platform documentation.
+
+==== Lead-in link sentence style for non-xref situations
+
+For situations where you cannot use the `xref` attribute, such as referencing a module within a module, ensure that you adhere to the following guidance for lead-in link sentences.
+
+* Use the following format when pointing to a section in the same document: `For more information, see "<section_name>".`
+* Wrap the `<section_name>` string in quotation marks. The section name can be the title of a module or an assembly.
+* Do not state "in the Additional Resources section" or other variants. Mentioning the `<section_name>` is enough to prompt a documentation user to find the section in the same document.
+
+==== Lead-in link sentence style for the xref attribute
+
+For situations where you can use the `xref` attribute, adhere to the following guidance:
+
+* When pointing to a section that exists in the same documentation set, create an _Additional resources_ section in the assembly file and then place the xref as a bulleted item under this section. For example:
++
+[source,text]
+----
+[role="_additional-resources"]
+.Additional resources
+* xref:../../../installing/installing_vsphere/installation-config-parameters-vsphere.adoc#installation-config-parameters-vsphere[Installation configuration parameters]
+----
+* When pointing to a link in a section that exists in another customer product document, use the `For more information on <topic>, see "<section_name>" in _<document_name>_.` format. Note the quotation marks around `<section_name>` and the italics around `<document_name>`.
```

Comment:
Could this be formed as a block example (like the previous bullet) instead of inline code/example?
```suggestion
* When pointing to a link in a section that exists in another customer product document, use the following format:
+
[source,text]
----
For more information on <topic>, see "<section_name>" in _<document_name>_.
----
+
Note the quotation marks around `<section_name>` and the italics around `<document_name>`.
```
This also has the benefit of the underscores not rendering in-line as italics when viewing the rendered GitHub Markdown (as it's doing currently), and they remain viewable in the source block:
<img width="834" alt="image" src="https://github.com/openshift/openshift-docs/assets/3442316/854d76f7-699f-483b-9365-89125be3d8f6">